### ISSUE TITLE
Package already published - Sinopia

### ISSUE THREAD

**Issue by asaf** [drone-plugins/drone-npm#8]
Hey,

I'm trying to publish some package, but it always says _Package already published_

output:

```bash
Requesting https://myrepo:4444/mypkg/0.0.4
Package already published
```

This is weird, because executing _curl_ with:

`curl https://myrepo:4444/mypkg/0.0.4 -v -u myUser:myPass`

The result is:
```bash
...
< HTTP/1.1 404 Not Found
< X-Powered-By: Sinopia/1.4.0
...
{
  "error": "version not found: 0.0.4"
}
```

So i'm not really sure why [this condition](https://github.com/drone-plugins/drone-npm/blob/master/main.go#L203) is set to true

(I tried cloning code and running `make deps build` but go started go crazy on my OSX so I'm not sure what to do next without digging too much)

Thanks :-)

Asaf.

**Comment by asaf** [comment]
@tboerger Any chance helping out how to debug the problem myself?

I cloned the project into /home/user/go/drone-npm

I set `$GOPATH=/home/user/go/drone-npm`

Running `make deps build`

I get:

```bash
package github.com/drone/drone-go/plugin: cannot find package "github.com/drone/drone-go/plugin" in any of:
	/usr/local/go/src/github.com/drone/drone-go/plugin (from $GOROOT)
	/home/user/go/drone-npm/src/github.com/drone/drone-go/plugin (from $GOPATH)
make: *** [deps] Error 1
```

What am I missing?

Thanks.

**Comment by donny-dont** [comment]
@asaf I'm not sure what the problem is because I run sinopia here and actually developed this plugin with sinopia. I added some additional logging of things in the 0.5 update so maybe that will help.

The only guess I have is that it could be rejecting you because of authentication. Do you have always_auth on?

**Comment by asaf** [comment]
@donny-dont I suspect the problem is SSL,
I have a self signed certificate, and probably the plugin fails due to that on [line](https://github.com/drone-plugins/drone-npm/blob/master/main.go#L197),

I can't debug since this plugin doesn't get built (there's no such thing _github.com/drone/drone-go/plugin_, see [this issue](https://github.com/drone/drone-go/issues/27)

**Comment by donny-dont** [comment]
I just did an update for 0.5 if you want to try and run that there are instructions on #6 . It adds some additional logging.

**Comment by asaf** [comment]
@donny-dont I guess I need Drone 0.5 and everything is going to break :-)
I'll try, thanks,

If I succeed, do you mind if I try to add support for server side certificate ?

**Comment by donny-dont** [comment]
No the nice thing about the Drone 0.5 stuff is that you can simply run the container to test it. That's why I was pointing you at it. I'm ok with backporting things just need to know what exactly your issue is. It looks like SSL stuff so I can add something to disable those checks. I'm working on this stuff today so now would be a good time to take a look :smile_cat:

**Comment by asaf** [comment]
@donny-dont Awesome, are you in some gitter room so we can move things quickly?

**Comment by donny-dont** [comment]
@asaf yep I pinged you.

**Comment by donny-dont** [comment]
Fixed in the 0.5 implementation
